docs/getting-started/tutorial-visualizing.asciidoc

```diff
@@ -141,11 +140,15 @@ You haven't defined any buckets yet, so the visualization is a map of the world.
 
 . Map the geo coordinates from the log files.
 
-.. In the *Buckets* pane, click *Add > Geo coordinates*.
-.. Set *Aggregation* to *Geohash*.
-.. Set *Field* to *geo.coordinates*.
+.. Click *Add layer* button.
+.. Click the *Grid aggregation* data source.
+.. Set *Index pattern* to *logstash*.
+.. Click the *Add layer* button.
 
-. Click *Apply changes* image:images/apply-changes-button.png[].
+. Set the vector style.
```

Comment:
The text I see in the UI is "Layer style".  Would it be better to say "Set the layer style"?